README.md

```diff
@@ -33,6 +42,21 @@ cmake ..
 make -j4
 ```
 
+## Building on MacOS M1
+
+```
+mkdir build
+cd build
+cmake .. -DCMAKE_C_COMPILER=/opt/homebrew/bin/gcc-12 -DCMAKE_CXX_COMPILER=/opt/homebrew/bin/g++-12
+cmake --build .
+```
+
+Otherwise it defaults to Clang toolset, which as of version:  
+
+Apple clang version 14.0.0 (clang-1400.0.29.202)  
+
+is incapable of compiling the library.
```

Comment:
Can you provide more details about this?

Reply:
Are you able to see that build log:

https://github.com/doodeck/libnetflow9/actions/runs/4681045806/jobs/8293144058

?
CMake Error at CMakeLists.txt:75 (message):
  Could not find required <memory_resource> or <experimental/memory_resource>
  headers

I guess the library was designed and build on top of gcc toolset. What is causing problems here is probably CLang toolset. Actually they might have fixed it for the future releases:

https://reviews.llvm.org/D89057